x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -200,13 +205,23 @@ const transformControlColumns = ({
         setCellProps,
       }: EuiDataGridCellValueElementProps) => {
         addBuildingBlockStyle(data[rowIndex].ecs, theme, setCellProps);
+        let disabled = false;
+        if (columnId === 'checkbox-control-column' && hasAlertsCrudPermissions != null) {
+          const alertConsumers =
+            data[rowIndex].data.find((d) => d.field === ALERT_RULE_CONSUMER)?.value ?? [];
+          disabled = alertConsumers.reduce<boolean>(
```

Comment:
NIT: maybe it would be easier to read using _some_:
`disabled = alertConsumers.some((consumer) => !hasAlertsCrudPermissions(consumer));`